Demo/AuthenticatorDemo/WKWebView+UserAgent.swift

```diff
@@ -0,0 +1,11 @@
+import WebKit
+
+extension WKWebView {
+
+    static var userAgent: String {
+        guard let userAgent = Self().value(forKey: "_userAgent") as? String, userAgent.isEmpty == false else {
+            return ""
+        }
+        return userAgent
+    }
+}
```

Comment:
Would it be useful to move this to the framework to make it available to consumers?

If so, would it be reasonable to keep the `""` fallback under the assumption that we'll always have a user agent from `WKWebView`, or should we return `String?` or make it `throw`, just in case?

Reply:
I'm leaning towards not exposing this as API, at least not this implementation. Creating a `WKWebView` to get a user agent string seems a little bit too expensive to me. Not to mention it uses a private API here. Unless the backend HTTP API requires a "browser like" user agent.